sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/AstyanaxDataReaderDAO.java

```diff
@@ -594,6 +598,15 @@ public String getPlacementCluster(String placementName) {
     @Override
     public Iterator<MultiTableScanResult> multiTableScan(final MultiTableScanOptions query, final TableSet tables,
                                                          final LimitCounter limit, final ReadConsistency consistency) {
+        return multiTableScanWithCutoffTime(query, tables, limit, consistency, null);
+    }
+
+    @Override
+    public Iterator<MultiTableScanResult> multiTableScanWithCutoffTime(final MultiTableScanOptions query, final TableSet tables,
+                                                                       final LimitCounter limit, final ReadConsistency consistency, UUID timeUUID) {
+        if (timeUUID != null && timeUUID.version() != 1) {
+            throw new IllegalArgumentException(); // must be a time uuid
+        }
```

Comment:
As previously noted, changing to a `Date` or `DateTime` removes the need for this check.